WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -614,7 +643,11 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             requestHTMLReason = nil // clear the reason
             switch reason {
             case .publish:
-                handlePublishButtonTap()
+                if editorHasContent(title: title, content: html) {
```

Comment:
I'm considering merging this PR with only the changes on WP-iOS to avoid the issue of saving an empty post.
Then do a separate PR in GB-mobile to improve the check of content using the GB API's and updating the content info APIs. What do you think @etoledom ?

Reply:
Yeah sounds good! Let's fix the main issue now and improve with more time on a separated PR 👍